src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -159,4 +159,12 @@ trait MarathonConf extends ScallopConf with ZookeeperConf with IterativeOfferMat
     descr = "Mesos Authentication Secret",
     noshort = true
   )
+
+  //Internal settings, that are not intended for external use
+  lazy val internalStoreBackend = opt[String]("internal_store_backend",
+    descr = "The backend storage system to use. One of zk, mesos_zk",
+    hidden = true,
+    validate = Set("zk", "mesos_zk").contains,
```

Comment:
It would be occasionally useful to also allow `in_memory`. For example, to test whether persistence is our bottleneck.